### DISCUSSION TITLE
I choosed tabler as theme, but not config created

### DISCUSSION THREAD

**Discussion by realtebo** [Laravel-Backpack/community-forum#556] [General] [answered]
I cannot find in the doc where to change layout of new tabler theme

I see this output

```
 STEP 2  Installing a Theme.

  │ Choose your admin UI, depending on your project and preferences.

  Install the Tabler theme ............................................................................................... DONE
  Publish theme config file .............................................................................................. DONE
  Use theme as view namespace in config/backpack/ui.php .................................................................. DONE
```

But this is my config folder content

![image](https://github.com/Laravel-Backpack/community-forum/assets/1268699/395b6ea2-68a0-4290-882e-72110bb20d3f)

I was expecting something like https://github.com/Laravel-Backpack/theme-tabler/blob/main/config/theme-tabler.php 

What have I done wrong?

**Comment by pxpm** [comment, marked as answer]
Hey @realtebo thanks for the report.

I can confirm the issue and submitted the PR to fix it: https://github.com/Laravel-Backpack/CRUD/pull/5190

You have done nothing wrong, we did 🙃 

In the meanwhile you can manually publish the config file with: `php artisan vendor:publish --tag="theme-tabler-config"`

Cheers
